.github/workflows/build_tests.yaml

```diff
@@ -49,31 +53,34 @@ jobs:
       run: |
         gcloud config set compute/zone us-east4-a
         gcloud config get compute/zone
+    - name: Install xpk
```

Comment:
Oh perfect I was looking for this in the previous PR. I think we want to test both flows with python3 xpk.py and xpk (through pip)?

Maybe a build test that checks that [xpk.py / xpk] --help can be run is a good check for both flows in the build test, the rest of the build test can use just one of them. We could add a nightly test that tests our exhaustive command list?

Reply:
Please see my comment below. I believe it's easier to maintain just xpk through pip, and stop supporting using it through python interpreter - unless there are some cons I'm not aware of (maybe some users already use it with the python interpreter and it would break their workflows(?))